rfcs/0125-bootspec.md

```diff
@@ -110,11 +110,13 @@ Using the following JSON:
 {
   // Toplevel key describing the version of the specification used in the document
   "v1": {
+    // System for which the bootspec has been built, useful for architecture-dependent removable device fallback path
+    "system": "x86_64-linux",
```

Comment:
I think we should also add `(Required) ...` to all the other fields, to make it extremely obvious which fields are optional, and which are required for a compliant implementation.